net/net-snmp/files/snmpd.init

```diff
@@ -336,7 +336,7 @@ start_service() {
 	append_parm trapsess trapsess trapsess
 
 	procd_set_param command $PROG -Lf /dev/null -f -r -p "$pid_file"
-	procd_set_param file $CONFIGFILE
```

Comment:
Why is this removed? This changes reload behaviour, has nothing to do with the commit subject nor is it explained in the commit message.

Reply:
I just saw that I also have to remove this [line](https://github.com/openwrt/packages/blob/master/net/net-snmp/Makefile#L270) from the Makefile install target, when I add the procd line `procd_append_param command -C -c $CONFIGFILE` to load the generated config file.